kibble/api/prices.go

```diff
@@ -126,11 +126,31 @@ func processPrices(details prices, site *models.Site, itemIndex models.ItemIndex
 		}
 	}
 
+	// loop over the retrieved list of films that belong to plans
+	// assign a slice of maps to each film e.g. [{"plan/123" : prices }, ...]
+	for _, filmPlans := range details.Plans {
+		if film, ok := site.Films.FindFilmBySlug(filmPlans.Item); ok {
+
+			filmPlanPrices := make(map[string]models.PriceCollection, 0)
```

Comment:
Can this be `make(PlanPriceCollection, 0)` given PlanPriceCollection is a map of strings to PriceCollection?

Reply:
changed